app/views/audit_mailer/summary.text.erb

```diff
@@ -0,0 +1,15 @@
+<%= _("Foreman audit summary") %>
+=================================
+<% if @audits.size > 0 %>
+  <%= n_('Displaying %{num_audits} of %{total_audits} audits',
+         'Displaying %{num_audits} of %{total_audits} audits', @count) % {:num_audits => @limit, :total_audits => @count} %>
+  <% @audits.each do |audit| %>
+    <%= audit.username %>
+    ( <%= audit.remote_address %> )
```

Comment:
These lines (8-10) should probably be on the same line as the username so it reads more correctly, it's a bit disjointed spread over four lines when it's usually shown on one.